packages/core/project.json

```diff
@@ -0,0 +1,39 @@
+{
+  "root": "packages/core",
+  "sourceRoot": "packages/core/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/core",
+        "main": "packages/core/src/index.ts",
+        "tsConfig": "packages/core/tsconfig.build.json",
```

Comment:
How come that `tsconfig.lib.json` became `tsconfig.build.json`? Please don't rename autogenerated files to avoid such mistakes if it is not necessary

Reply:
Changes was reverted